Resources/doc/getting_started/the_form_view.rst

```diff
@@ -110,7 +108,7 @@ as choice.
     }
 .. note::
 
-    '`property`_' field is not supported by Symfony >= 2.7. You should use `choice_label`_ instead.
+    The `property`_ option is not supported by Symfony >= 2.7. You should use `choice_label`_ instead.
```

Comment:
this is a link @Soullivaneuh this should work as before.

Check the Symfony.com website here:
http://symfony.com/doc/master/bundles/SonataAdminBundle/getting_started/the_form_view.html#adding-fields-that-reference-other-models